src/shared/views/components/dataset/KeyInfo.jsx

```diff
@@ -30,6 +31,27 @@ export default function KeyInfo(props) {
     return <T>dataset_view.key_information.next_update_missing</T>;
   }
 
+  function PeriodCovered(props) {
+    if (props.timePeriod.start && props.timePeriod.end) {
+      return (
+        <div id="time-period" className="govuk-summary-list__row">
+          <dt className="govuk-summary-list__key">
+            <T>dataset_view.key_information.time_covered</T>
+          </dt>
+          <dd className="govuk-summary-list__value">
+            <T
+              start={dateFormat(props.timePeriod.start, 'MMMM yyyy', props.lang)}
+              end={dateFormat(props.timePeriod.end, 'MMMM yyyy', props.lang)}
+            >
+              dataset_view.key_information.time_period
+            </T>
+          </dd>
+        </div>
+      );
+    }
+    return <div></div>;
```

Comment:
Rather than returning an empty div, just return `false` (or `null`, but false is better apparently) and React won't render anything. e.g.

```
if (!props.timePeriod.start || !props.timePeriod.end) {
  return false;
}

return <actual-component>;
```